spec/models/taxon_name_spec.rb

```diff
@@ -79,6 +79,28 @@
     expect( TaxonName.make!( lexicon: " Foo" ).lexicon ).to eq "Foo"
   end
 
+  it "should not be valid with a non-English translation of a lexicon" do
+    tn = TaxonName.make(lexicon: "简体中文", name: "common")
```

Comment:
Would be cool to pull this from `t()` instead, but not sure how to get i18n into specs easily.

Reply:
I *think* it should work: https://github.com/inaturalist/inaturalist/blob/main/spec/models/observation_spec.rb#L438